src/conditions/check_fields.rs

```diff
@@ -15,6 +16,8 @@ pub enum CheckFieldsPredicateArg {
     #[derivative(Debug = "transparent")]
     String(String),
     #[derivative(Debug = "transparent")]
+    VecString(Vec<String>),
+    #[derivative(Debug = "transparent")]
```

Comment:
Did you give any consideration to lists of non-strings (like numbers)? I realize it will explode the variants if all possibilities are included.

Reply:
I didn't and that's a fair point. Hmmmm. Guess it didn't occur to me since we aren't using the number for filtering.

Not sure how useful it is to specify a list of numbers. Maybe giving a range is more useful? Not sure on this.

Reply:
Yeah, I'm not sure either. We'll address that use case separately if it comes up unless you have some bright ideas for it.